docs/en/stack/ml/anomaly-detection/model-snapshots.asciidoc

```diff
@@ -8,21 +8,28 @@ accomplished by generating models of your data.
 
 To ensure resilience in the event of a system failure, snapshots of the {ml}
 model for each {anomaly-job} are saved to an internal index within the {es}
-cluster. By default, snapshots are captured approximately every 3 to 4 hours and 
-retained for one day (twenty-four hours). The amount of time necessary to
-save these snapshots is proportional to the size of the model in memory.
+cluster. The amount of time necessary to save these snapshots is proportional to
+the size of the model in memory. By default, snapshots are captured
+approximately every 3 to 4 hours. You can change this interval
+(`background_persist_interval`) when you create or update a job.
 
-You can use the {ref}/ml-update-job.html[update {anomaly-jobs} API] to change
-the interval (`background_persist_interval`) and retention
-(`model_snapshot_retention_days`) of these snapshots.
+To reduce the number of snapshots consuming space on your cluster, at the end of
+each day, old snapshots are automatically deleted. By default, if there are
```

Comment:
Do we need to cover how age is calculated here too? For example:
```suggestion
each day, old snapshots are automatically deleted. The age of each snapshot is calculated relative to the timestamp of the most recent snapshot. By default, if there are
```

Reply:
Yes, we probably should, because it's natural to think of "age" as being relative to now, and that's not the case here.

Reply:
Thanks, I've implemented those changes.